data/systemd/snapd.run-from-snap

```diff
@@ -34,6 +34,8 @@ case $CMD in
             echo "cannot run without the prefix for the snapd snap"
             exit 1
         fi
+        echo "Add the bind mount of the snapd tooling"
+        mount -o bind "$PREFIX"/usr/lib/snapd /usr/lib/snapd
```

Comment:
Could this be a mount unit instead?

Reply:
What is the advantage here? It seems like its more complex in terms of state to manage, i.e. the script needs to write one, clean out stale ones etc?

Reply:
It is more discoverable and explicit (less magic is good).

Question: does systemd shut down properly with this bind mount in place?

Reply:
I think one could be generated and dumped into `/run/systemd/system`. Then the units that are generated just below could have proper `Requires=<that>.mount` snipped added.